client/my-sites/people/people-list-item/index.jsx

```diff
@@ -44,40 +45,58 @@ class PeopleListItem extends React.PureComponent {
 		);
 	};
 
+	getCardLink = () => {
+		const { invite, site, type, user } = this.props;
+		const editLink = this.canLinkToProfile() && `/people/edit/${ site.slug }/${ user.login }`;
+		const inviteLink = invite && `/people/invites/${ site.slug }/${ invite.invite_key }`;
+
+		return type === 'invite' ? inviteLink : editLink;
+	};
+
+	renderInviteStatus = () => {
+		const { invite, translate } = this.props;
+		const { is_pending: isPending } = invite;
```

Comment:
We can get rid of this awkwardness by transforming the invites objects we receive from the API as they are inserted into Redux state.  This is a good idea for other reasons too:  https://github.com/Automattic/wp-calypso/pull/21505#discussion_r161935011

I don't have a strong preference whether this should be done here or in a separate PR.  Probably a separate PR so that we can keep moving.